CMakeLists.txt

```diff
@@ -39,8 +40,9 @@ endif()
 #
 # tests
 #
-if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME AND BUILD_TESTING)
+if((${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME} OR ${PROJECT_NAME}_BUILD_TESTING) AND BUILD_TESTING)
     include(CTest)
+    enable_testing()
```

Comment:
https://cmake.org/cmake/help/latest/command/enable_testing.html

should not be necessary as we're requiring `BUILD_TESTING`

Reply:
removed